python/lsst/analysis/tools/interfaces/_task.py

```diff
@@ -212,7 +213,8 @@ class AnalysisBaseConfig(PipelineTaskConfig, pipelineConnections=AnalysisBaseCon
         doc="Which time stamp should be used as the reference timestamp for a "
         "metric in a time series database, valid values are; "
         "reference_package_timestamp, run_timestamp, current_timestamp, "
-        "and dataset_timestamp",
+        "dataset_timestamp and explicit_timestamp:datetime where datetime is"
```

Comment:
Repeated 'is' on the line below. Make sure not to forget the trailing whitespace on correction.